packages/studio-base/src/panels/index.ts

```diff
@@ -87,14 +87,14 @@ const builtin: PanelInfo[] = [
   {
     title: "Publish",
     type: "Publish",
-    description: "Publish data over a life connection.",
+    description: "Publish messages to the data source.",
     help: PublishHelp,
     module: async () => await import("./Publish"),
   },
   {
     title: "Raw Messages",
     type: "RawMessages",
-    description: "Display topic messages as JSON.",
+    description: "Inspect messages on a topic.",
```

Comment:
"on a topic" is basically implied for every panel.
```suggestion
    description: "View message data in its original, raw form.",
```

Reply:
`data in its original, raw form` sounds awkward. Do we think `inspect` is not obvious? Maybe it isn't and I am just used to it cause of the tools I use.

I'm still a fan of `Inspect messages on a topic.`

Reply:
Changed to `Inspect or find the diff between messages.`